doc/releases/release-notes-3.1.rst

```diff
@@ -619,8 +619,167 @@ Networking
+    relying on this will need updates.
+
+  * Bindings for ADC controller nodes can now use a child binding to specify
+    the initial configuration of individual channels in devicetree. See pull
+    request `43030 <https://github.com/zephyrproject-rtos/zephyr/pull/43030>`_
+    for details.
+
+  * New bindings for the following compatible properties were added:
+
+    * :dtcompatible:`arduino-nano-header-r3`
+    * :dtcompatible:`arm,cortex-r52`
+    * :dtcompatible:`atmel,sam-rstc`
+    * :dtcompatible:`can-transceiver-gpio` (see also :ref:`devicetree-can-api`)
+    * :dtcompatible:`gd,gd32-spi`
+    * :dtcompatible:`hynitron,cst816s`
+    * :dtcompatible:`intel,cavs-gpdma`
+    * :dtcompatible:`intel,cavs-hda-host-in` and :dtcompatible:`intel,cavs-hda-host-out`
+    * :dtcompatible:`intel,cavs-hda-link-in` and :dtcompatible:`intel,cavs-hda-link-out`
+    * :dtcompatible:`intel,ssp-dai`
+    * :dtcompatible:`intel,ssp-sspbase`
+    * :dtcompatible:`invensense,icm42670`
+    * :dtcompatible:`ite,enhance-i2c`
+    * :dtcompatible:`ite,it8xxx2-vcmp`
+    * :dtcompatible:`ite,it8xxx2-wuc` and :dtcompatible:`ite,it8xxx2-wuc-map`
+    * :dtcompatible:`ite,peci-it8xxx2`
+    * :dtcompatible:`maxim,max31875`
+    * :dtcompatible:`microchip,cap1203`
+    * :dtcompatible:`microchip,mcp4728`
+    * :dtcompatible:`microchip,mpfs-qspi`
+    * :dtcompatible:`microchip,xec-bbram`
+    * :dtcompatible:`motorola,mc146818`
+    * :dtcompatible:`nordic,nrf-acl`
+    * :dtcompatible:`nordic,nrf-bprot`
+    * :dtcompatible:`nordic,nrf-ccm`
+    * :dtcompatible:`nordic,nrf-comp`
+    * :dtcompatible:`nordic,nrf-ctrlapperi`
+    * :dtcompatible:`nordic,nrf-dcnf`
+    * :dtcompatible:`nordic,nrf-gpio-forwarder`
+    * :dtcompatible:`nordic,nrf-lpcomp`
+    * :dtcompatible:`nordic,nrf-mpu`
+    * :dtcompatible:`nordic,nrf-mutex`
+    * :dtcompatible:`nordic,nrf-mwu`
+    * :dtcompatible:`nordic,nrf-nfct`
+    * :dtcompatible:`nordic,nrf-oscillators`
+    * :dtcompatible:`nordic,nrf-ppi`
+    * :dtcompatible:`nordic,nrf-reset`
+    * :dtcompatible:`nordic,nrf-swi`
+    * :dtcompatible:`nordic,nrf-usbreg`
+    * :dtcompatible:`nuvoton,adc-cmp`
+    * :dtcompatible:`nxp,imx-mipi-dsi`
+    * :dtcompatible:`nxp,imx-qtmr`
+    * :dtcompatible:`nxp,imx-tmr`
+    * :dtcompatible:`raspberrypi,pico-reset`
+    * :dtcompatible:`raspberrypi,pico-usbd`
+    * :dtcompatible:`raydium,rm68200`
+    * :dtcompatible:`riscv,sifive-e31`, :dtcompatible:`riscv,sifive-e51`,
+      and :dtcompatible:`riscv,sifive-s7` CPU bindings
+    * :dtcompatible:`seeed,grove-lcd-rgb`
+    * :dtcompatible:`st,lsm6dso32`
+    * :dtcompatible:`st,stm32-clock-mux`
+    * :dtcompatible:`st,stm32-fmc-nor-psram`
+    * :dtcompatible:`st,stm32-lse-clock`
+    * :dtcompatible:`st,stm32-ltdc`
+    * :dtcompatible:`st,stm32-ospi` and :dtcompatible:`st,stm32-ospi-nor`
+    * :dtcompatible:`st,stm32h7-fmc`
+    * TI ADS ADCs: :dtcompatible:`ti,ads1013`, :dtcompatible:`ti,ads1015`,
```

Comment:
nit: there is also the :dtcompatible:`ti,ads1014`

Reply:
Nice catch. Fixed, thanks.